tracer.go

```diff
@@ -151,9 +153,8 @@ func (t *tracer) startSpanWithOptions(
 	operationName string,
 	options opentracing.StartSpanOptions,
 ) opentracing.Span {
-	startTime := options.StartTime
-	if startTime.IsZero() {
-		startTime = t.timeNow()
+	if options.StartTime.IsZero() {
```

Comment:
A span must have a start time. But `options.startTime` is an *optional* parameter, we do not expect most users to set it.